rasa/core/events/__init__.py

```diff
@@ -77,7 +87,7 @@ def md_format_message(
     message_from_md = entities_parser.parse_training_example(text, intent)
```

Comment:
optional: imo doesn't make sense that this function lives in the events module. Should be somewhere in `TrainingData` or `TrainingDataWriter`